torchgen/model.py

```diff
@@ -672,7 +672,7 @@ def from_yaml(
             # table is NOT semantics preserving
             assert structured_delegate or dispatch.keys() != {
                 DispatchKey.CompositeImplicitAutograd
-            }, (
+            } or dispatch[DispatchKey.CompositeImplicitAutograd].supports_symint(), (
```

Comment:
This needs to make it upstream

Reply:
I'm adding it as a part of the stack to symintify embedding